vignettes/Using_rerddap.Rmd

```diff
@@ -0,0 +1,671 @@
+---
+title: "Using rerddap to Access Data from ERDDAP Servers"
+author: "Roy Mendelssohn and Scott Chamberlain"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{rerddapVignette}
```

Comment:
Change the `VignetteIndexEntry` to something a little more informative, the one for the other vignette is `rerddap introduction`